src/App.js

```diff
@@ -1,15 +1,32 @@
 import { BrowserRouter } from 'react-router-dom';
 
 import './App.css';
-import Register from './components/Register';
-import Login from './components/Login/Login';
+// import Register from './components/Register';
+// import Login from './components/Login/Login';
+
+import { Box } from '@chakra-ui/react';
+
+import BasicInfo from './components/My Profile page/BasicInfo';
+import Security from './components/My Profile page/Security';
```

Comment:
Please always name folders this way (no spaces, all lower case)

```suggestion
import BasicInfo from './components/my-profile-page/BasicInfo';
import Security from './components/my-profile-page/Security';
```